pkg/config/config.go

```diff
@@ -124,6 +124,8 @@ const (
 const (
 	// DefaultIrqBalanceConfigFile default irqbalance service configuration file path
 	DefaultIrqBalanceConfigFile = "/etc/sysconfig/irqbalance"
+	// DefaultIrqBalanceConfigRestoreFile contains the banned cpu mask configuration to restore. Name due to backward compatibility.
+	DefaultIrqBalanceConfigRestoreFile = "/etc/sysconfig/orig_irq_banned_cpus"
```

Comment:
I would expect this config option would mean we remove the hard-coded one in `internal/runtimehandlerhooks/high_performance_hooks.go`

Reply:
also, given the name of the file, I wouldn't expect it to be a restore file. Is this file's name/format documented anywhere?

Reply:
I think this PR provides an alternative param for the user to have more control over the restore process while keeping the already working way to do it, so it will keep the exact same behaviour if the user do not give the new option. 
At least that is how I have understood the change but I could be wrong.

Reply:
That's true. We keep the filename originally added [back in time](https://github.com/cri-o/cri-o/pull/4441) for full backward compatibility.